health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FileComponent.js

```
@@ -44,15 +44,14 @@ const FileComponent = ({ title, fileName, status, auditDetails, editHandler, del
         }
     };
-
     return (
         <Card type="secondary">
             <div className="dm-parent-container" style={{ background: "#FAFAFA", margin: "0" }}>
                 <div
                     className="dm-uploaded-file-container-sub"
                     style={{ marginLeft: "-1rem" }}
                 >
-                    <div onClick={async () => await handleFilePreview(rowDetails?.fileStoreId)}>
+                    <div onClick={async () => await handleFilePreview(rowDetails?.fileStoreId || rowDetails?.filestoreId)}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)